download-directory/src/main/java/org/reactome/release/downloadDirectory/Main.java

```diff
@@ -41,8 +41,7 @@ public static void main(String[] args) throws Exception {
 		int port = Integer.valueOf(props.getProperty("release.database.port"));
 		String releaseNumber = props.getProperty("releaseNumber");
 		String releaseDirAbsolute = props.getProperty("absoluteReleaseDirectoryPath");
-		String releaseDownloadDir = props.getProperty("releaseDownloadDirectoryPath");
-		String releaseDownloadDirWithNumber = releaseDownloadDir + releaseNumber;
+		String releaseDownloadDirWithNumber = Paths.get("download_directory", releaseNumber).toString();
```

Comment:
Is this path supposed to be relative or does it need to be prefixed by the value in `releaseDirAbsolute`?

Reply:
Whoops, great catch